py/torch_tensorrt/fx/converters/aten_ops_converters.py

```diff
@@ -465,6 +502,27 @@ def aten_ops_sym_numel(
     return reduce_layer.get_output(0)
 
 
+@tensorrt_converter(torch.ops.aten.slice.Tensor)
+def aten_ops_slice(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return slice_op(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+        args[2],
+        args[3],
```

Comment:
For `args[3]`, as in the TorchScript `slice` converter, the `aten` opset sometimes uses `9223372036854775807` to represent the entire dimension. We might use a check like `args[3] if args[3] != 9223372036854775807 else args[0].shape[args[1]]` to deal with this case.

Reply:
Looking at this.